gems/moji/1.6/moji.rbs

```diff
@@ -0,0 +1,5 @@
+module Moji
+  def self.han_to_zen: (::String str, ?untyped tp) -> String
```

Comment:
As you said, these methods are defined as `module_function`s. How about using `self?` instead of `self`? `self?` represents module function feature.

https://github.com/ruby/rbs/blob/master/docs/syntax.md#method-definition

> self? method definition adds two methods: a public singleton method and a private instance method, which is equivalent to module_function in Ruby.